exporters/otlp/otlplog/otlploghttp/exporter.go

```diff
@@ -5,14 +5,16 @@ package otlploghttp // import "go.opentelemetry.io/otel/exporters/otlp/otlplog/o
 
 import (
 	"context"
+	"sync/atomic"
 
 	"go.opentelemetry.io/otel/sdk/log"
 )
 
 // Exporter is a OpenTelemetry log Exporter. It transports log data encoded as
 // OTLP protobufs using HTTP.
 type Exporter struct {
-	// TODO: implement.
+	client  atomic.Pointer[client]
+	stopped atomic.Bool
```

Comment:
I know this is late but... As far as I remember the exporters do not have to be concurrent safe as the processors handle synchronization. This is also why `Exporter` interface does not tell anything about concurrency as opposed to `Processor` interface.